runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpTrackerPlugin.java

```diff
@@ -75,6 +77,7 @@
 	private ScheduledExecutorService executor;
 
 	private NavigationButton navButton;
+	@Getter(AccessLevel.PACKAGE)
```

Comment:
Dont expose everything. Rather, try to figure out how to do this with current access levels